heron/instance/src/java/com/twitter/heron/instance/Slave.java

```diff
@@ -66,6 +66,12 @@
   private State<Serializable, Serializable> instanceState;
   private boolean isStatefulProcessingStarted;
 
+  private static final int INSTANCE_INTERNAL_BOLT_READ_QUEUE_CAPACITY = 128;
```

Comment:
@jerrypeng - comment about how these values are being chosen will be helpful. Are these values configurable by the user?

Reply:
@kramasamy I discussed with @srkukarni and he recommended we just use some hard coded values since these values for the queue are going to be transient anyways

Reply:
@jerrypeng - that is fine. please just add a comment saying that we are using arbitrary values since it is going to transient. Later when somebody reads the code, they might not be able to know the context.

Reply:
The values used are default values chosen from the heron_internals.yaml file

Reply:
@kramasamy sure I can add some comments